src/sentry/api/endpoints/relay/project_configs.py

```diff
@@ -67,12 +53,17 @@ def _post(self, request: Request):
         version = request.GET.get("version") or "1"
         set_tag("relay_protocol_version", version)
 
+        if version == "4":
+            if request.relay_request_data.get("global"):
+                metrics.incr("relay.project_configs.global.fetched", tags={"version": version})
+                response["global"] = get_global_config()
```

Comment:
The prior implementation returned `global_config`, now it is `global`. In the PR description, could you please link the implementation in Relay (either master or open PR) that this corresponds to, so we can check if it matches?